scripts/ci-footprint-and-performance/setup/100-reboot.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -xeuo pipefail
+export PS4='+ $(date "+%T.%N") ${BASH_SOURCE}:$LINENO \011'
+
+sudo reboot now
```

Comment:
Should this script include the code for waiting until the instance is accessible?
It's probably elsewhere, but that depends on the order of scripts, assuming the reboot is always the last one.

Reply:
All of these scripts are executed on the host that's being rebooted. Waiting for host up is in the CI step

Reply:
Of course. That makes sense.